### ISSUE TITLE
GCM core not working or somehow linked to coworker's activity

### ISSUE THREAD

**Issue by DesBaril** [microsoft/Git-Credential-Manager-Core#523]
GCM Core version : 2.0.567+3047faf390

**Which Git host provider are you trying to connect to?**

* [X ] Azure DevOps

**Can you access the remote repository directly in the browser using the remote URL?**

* [X ] Yes
* [ ] No, I get a permission error
* [ ] No, for a different reason - please describe

---

**_[Azure DevOps only]_ What format is your remote URL?**

* [ ] Not applicable
* [ X ] https://dev.azure.com/`{org}`/...
* [ ] https://`{org}`@dev.azure.com/`{org}`/...
* [ ] https://`{org}`.visualstudio.com/...

**_[Azure DevOps only]_ If the account picker shows more than one identity as you authenticate, check that you selected the same one that has access on the web.**

* [ X ] Not applicable
* [ ] I only see one identity
* [ ] I checked each identity and none worked

---

**Expected behavior**

When executing operations on the remote repository (aka git push/pull/fetch etc.) it should work and do so with my account.

**Actual behavior**
I'm on macos
First, after installing git-credential-manager-core, I never got the pop-up to identify
Second, our organisation is using PAT generated in Azure Dev Ops and linked to our account. We take the PAT, transfer it in base64, and then place it in our user level .npmrc file.
Third, when I push commits, the commits are in my name, but they appear as pushed by a coworker in Azure. Also, I being signed in does not change a thing, I can only do operations when this same coworker is signed in. 

We have all followed the same processus to create those pat, placed them at the same place, are all on macos but I'm the only one with this problem.
I have fully uninstalled git and git-credential-manager-core and cleared their cache many times. I have deleted and create multiple new PAT, changed my azure password but nothing has worked.

TLDR: My gcm core does not work as expected. I can only do operations on the repo when my coworker is signed in.
